Changelog.md

```diff
@@ -18,6 +18,7 @@
 ## Features
 * Add links to the aspects and followed tags pages on mobile [#7265](https://github.com/diaspora/diaspora/pull/7265)
 * diaspora\* is now available in Gàidhlig, Occitan, and Schwiizerdütsch
+* admins cannot be blocked anymore [#7279](https://github.com/diaspora/diaspora/issues/7297)
```

Comment:
You can still use this PR to fix it another way, or you can close it yourself if you don't want to work at this issue anymore.